lib/testing/src/runtime/checked_mode.dart

```diff
@@ -33,10 +33,12 @@ bool _checkForCheckedMode() {
   Object sentinal = new Object();
   try {
     var i = 1 as dynamic;
-    String string = i;
+    _takeString(i);
```

Comment:
Ha, yes, that would be simpler. I'm not sure what the original code was about, but I tried to stay as close to it as possible.